docs/source/_json/image.json

```diff
@@ -20,14 +20,20 @@ content-type: application/json
   "exclude_from_nav": false, 
   "expires": null, 
   "image": {
-    "icon": "http://localhost:55001/plone/image/@@images/image/icon", 
-    "large": "http://localhost:55001/plone/image/@@images/image/large", 
-    "listing": "http://localhost:55001/plone/image/@@images/image/listing", 
-    "mini": "http://localhost:55001/plone/image/@@images/image/mini", 
-    "original": "http://localhost:55001/plone/image/@@images/image", 
-    "preview": "http://localhost:55001/plone/image/@@images/image/preview", 
-    "thumb": "http://localhost:55001/plone/image/@@images/image/thumb", 
-    "tile": "http://localhost:55001/plone/image/@@images/image/tile"
+    "content-type": "image/png", 
+    "download": "http://localhost:55001/plone/image/@@images/image", 
```

Comment:
I would only include either the "download" or the "original" link, because both point to the same url.